### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by PedroH-Neves**
Here I initialized the project with components and pages.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div>
```
Comment (Deepakdanger) [inline src/App.js]: - [ ] no need to add `div ` here. kindly remove it.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,202 @@
...
+
+- Clone the repository to your computer, or you download the zip file containing the repository files.
+
+- In order to use is possible to :
+
+- Open editor terminal
+
+- Digit "git clone https://github.com/PedroH-Neves/bookstore-react.git"
+
+- Then "cd bookstore-react"
```
Comment (Deepakdanger) [inline README.md]: - [ ] Here it should be `https://github.com/PedroH-Neves/book-store-react.git` & `cd book-store-react`.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,70 +1,202 @@
...
+<!-- Features -->
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Key Features <a name="key-features"></a>
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- **[]**
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- **[]**
```
Comment (Tufahel) [inline README.md:71]: - [ ] ***[OPTIONAL]*** Kindly don't keep this section empty as it is not a good practice. Add some `Key Features` related to the project.
- image

File: README.md [hunk 4]
```diff
@@ -1,70 +1,202 @@
...
+### Run tests
+
+To run tests, run the following command:
+
+- Run `npm start` to open the site
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
```
Comment (Tufahel) [inline README.md:140]: - [ ] ***[OPTIONAL]*** This sentence `You can deploy this project using:` is meaningless. You can write `N/A` or `You can deploy this project using Netlify/Heroku`. Kindly fix this issue.

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @PedroH-Neves :wave: 
 
Well done! 👍  Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Linter checks are passing.
:heavy_check_mark: Descriptive pull request.
:heavy_check_mark: Commit messages are professional.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**